src/modules/News/controller.ts

```diff
@@ -78,4 +78,45 @@ const getTopHeadlines = async (req: Request, res: Response) => {
   }
 };
 
-export default { getLatestNews, getTopHeadlines, getPersonalizedNews };
+const getSources = async (req: Request, res: Response) => {
+  const { language, country, category } = req.query;
+
+  try {
+    const sources = await fetchFromNewsAPI('/sources', {
+      language,
+      country,
+      category,
+    });
+    res.status(200).send(sources.sources);
```

Comment:
The .send(<customize response>) should be according to the requirement item 2.5: 

https://github.com/StartSteps-Digital-Education-GmbH/Course-Navigator/blob/main/Curriculum/05.%20Full-Stack%20TypeScript%20Development/NEWS_APP/02_Main_API_Refrence.md

Reply:
@regikov try to follow the guide, but if it's being done differently in other routes you can do it differently as well. The guide is like a suggestion, but you don't have to completely follow it if you think your solution is more logical.